packages/documentation/src/stories/foundation/accessibility/regulation.docs.mdx

```diff
@@ -0,0 +1,17 @@
+import { Meta, Source, Canvas } from '@storybook/blocks';
+import * as RegulationStories from './regulation.stories';
+
+<Meta of={RegulationStories} />
+
+# Accessibility
+
+<figure>
+  <blockquote class="blockquote" lang="de">
+    <p>Die Postdienste müssen so angeboten werden, dass Menschen mit Behinderungen
```

Comment:
Can we translate this to english? For example with the post translater or with deepl or some other online service.